### DISCUSSION TITLE
Would like to help with the project

### DISCUSSION THREAD

**Discussion by KtlTheBest** [fox0430/moe#1471] [General]
Fox0430-san, good time of the day.

I am currently taking a university course where we have to openly contribute to open-source projects. So I would like to contribute to this project in the span of 12 weeks, and maybe more after the course ends.

I didn't write in Nim before, but I am open to learning new things, know about Nim's features and metaprogramming abilities, as well as high-level and low-level interchanges.

It seems there are some features that need help. I can try to work on experimentation or implement needed features. Not sure how to do them yet, but fuzzy search and code over ssh sounds interesting (one may as well try to add synchronous editing).

Overall would like to hear from you soon and start working the project.

Best regards,
KtlTheBest

**Comment by fox0430** [comment]
@KtlTheBest 

Hello, no problem if you are really interested in this project.

This question is just out of curiosity.
I understand that this is a university project.
But why did you choose this project? There are other projects on Nim or text editors.

**Reply by KtlTheBest** [reply]
@fox0430 Good afternoon!

It is true, that there are many projects on Nim. But your project just caught my attention. That, and the fact that you have written most of the code along with your friend, it is amazing (for a lack of a better word). Somehow this dynamic makes me want to stay. I hope to learn a lot from this project and in turn help moe reach its potential.

O sewa ni narimasu. Yoroshiku o negai shimasu ~

**Reply by fox0430** [reply]
@KtlTheBest 

Ok, Thank you! welcome your PR and issues. Please feel free to contact me (Github, Discord, Email, etc...).
